python/ray/tune/tests/test_reporter_no_usage.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+import ray
+from ray.tune import Experiment
+
+
+class TuneServerSuite(unittest.TestCase):
+    def test_reporter_no_usage(self):
```

Comment:
Can you actually move this into `tune/tests/test_trial_runner.py` as a function of `TrainableFunctionApiTest`?